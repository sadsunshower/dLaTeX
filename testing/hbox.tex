\documentclass{article}

\begin{document}

Test: hbox

\framebox[\textwidth]{This box will be slightly too wide.}

\framebox[10\textwidth]{This box will be way too wide.}

\end{document}