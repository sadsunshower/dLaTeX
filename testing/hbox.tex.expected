This is dLaTeX version 0.1
{"messages": [], "error": false}
